tests/filecheck/mlir-conversion/with-mlir/dialects/memref/memref_ops_custom.mlir

```diff
@@ -19,7 +19,9 @@ memref.store %v0, %m[%i0, %i1] {"nontemporal" = true} : memref<2x3xi32>
 %v2 = memref.load %m[%i0, %i1] {"nontemporal" = false} : memref<2x3xi32>
 %v3 = memref.load %m[%i0, %i1] {"nontemporal" = true} : memref<2x3xi32>
 %r1 = memref.expand_shape %r [[0, 1], [2]] : memref<10x3xi32> into memref<5x2x3xi32>
-%r2 = memref.collapse_shape %r [[0, 1]] : memref<10x3xi32> into memref<30xi32> 
+%r2 = memref.collapse_shape %r [[0, 1]] : memref<10x3xi32> into memref<30xi32>
+
+%bla = tensor.empty() : tensor<2x3xf32>
```

Comment:
this should be in its own file (tests/filecheck/mlir-conversion/with-mlir/tensor/ops.mlir), I was rushing things